.github/workflows/deploy.yml

```diff
@@ -17,19 +17,29 @@ on:
           - dev
 
 jobs:
+  tag_version:
+    runs-on: ubuntu-latest
+    outputs:
+      version_to_tag: ${{ steps.tagging.outputs.tag_value }}
+    steps:
+      - id: tagging
+        run: |
+          echo "tag_value=$(echo '${{ github.ref }}' | sed -e 's,.*/\(.*\),\1,')" >> $GITHUB_OUTPUT
+
   paketo_build:
     permissions:
       packages: write
     uses: communitiesuk/funding-service-design-workflows/.github/workflows/package.yml@main
     with:
       assets_required: true
-      version_to_build: $(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+      version_to_build: ${{ needs.tag_version.outputs.version_to_tag }}
```

Comment:
This is correct, but you're missing something vital from the job...

Reply:
Pushed that change 👍

Reply:
Discussed in person - this `needs.tag_version.outputs` is referring to a 'needs' section, since it doesn't exist this would have ended up being blank